core/src/main/java/org/apache/carbondata/core/datastore/page/UnsafeFixLengthColumnPage.java

```diff
@@ -541,26 +539,6 @@ public long getPageLengthInBytes() {
     return totalLength;
   }
 
-  @Override public byte[] compress(Compressor compressor) throws MemoryException, IOException {
-    if (UnsafeMemoryManager.isOffHeap() && compressor.supportUnsafe()) {
-      // use raw compression and copy to byte[]
-      int inputSize = totalLength;
-      long compressedMaxSize = compressor.maxCompressedLength(inputSize);
-      MemoryBlock compressed =
-          UnsafeMemoryManager.allocateMemoryWithRetry(taskId, compressedMaxSize);
-      long outSize = compressor.rawCompress(baseOffset, inputSize, compressed.getBaseOffset());
```

Comment:
does it means that no more use unsafe compression? Because method `rawCompress` and `supportUnsafe` are only called by this method.